src/meinberlin/meinberlin/tests/acceptance/setup_test.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
Can change the script to python.

A missing setup_tests script does not lead to a protractor error, thus it could be added only to modules where its needed.